spanner/doc.go

```diff
@@ -300,12 +300,10 @@ of the transaction:
         }
         balance -= 10
         m := spanner.Update("Accounts", []string{"user", "balance"}, []interface{}{"alice", balance})
-        txn.BufferWrite([]*spanner.Mutation{m})
-
+        return txn.BufferWrite([]*spanner.Mutation{m})
```

Comment:
nit: could we move this line to after the comment. It does not feel right to have a comment after the return statement of a function.

Reply:
Yup, I actually I wondered if it's right. 
Then, fixed this with the similar code I referred to.

https://github.com/googleapis/google-cloud-go/pull/3803/commits/64546a29dd025fb6a75b2991c7cc38151e38bfd2